GestaoFinanceira/GestaoFinanceira/Views/FrmDashBoard.cs

```diff
@@ -17,27 +17,105 @@ public FrmDashBoard()
+            this.ctCategories.Series["Categories"].Points[1].Label = "15%";
+            this.ctCategories.Series["Categories"].Points.Add(3);
+            this.ctCategories.Series["Categories"].Points[2].LegendText = "Categoria C";
+            this.ctCategories.Series["Categories"].Points[2].SetValueXY(30, 30);
+            this.ctCategories.Series["Categories"].Points[2].Label = "35%";
+            this.ctCategories.Series["Categories"].Points.Add(4);
+            this.ctCategories.Series["Categories"].Points[3].LegendText = "Categoria D";
+            this.ctCategories.Series["Categories"].Points[3].SetValueXY(5, 5);
+            this.ctCategories.Series["Categories"].Points[3].Label = "5%";
+            this.ctCategories.Series["Categories"].Points.Add(5);
+            this.ctCategories.Series["Categories"].Points[4].LegendText = "Categoria E";
+            this.ctCategories.Series["Categories"].Points[4].SetValueXY(35, 35);
+            this.ctCategories.Series["Categories"].Points[4].Label = "35%";
+
+            this.ctBank.Series["Bank"].Points.Add(0);
+            this.ctBank.Series["Bank"].Points[0].LegendText = "Banco A";
+            this.ctBank.Series["Bank"].Points[0].SetValueXY(15, 15);
+            this.ctBank.Series["Bank"].Points[0].Label = "15%";
+            this.ctBank.Series["Bank"].Points.Add(1);
+            this.ctBank.Series["Bank"].Points[1].LegendText = "Banco B";
+            this.ctBank.Series["Bank"].Points[1].SetValueXY(15, 15);
+            this.ctBank.Series["Bank"].Points[1].Label = "15%";
+            this.ctBank.Series["Bank"].Points.Add(3);
+            this.ctBank.Series["Bank"].Points[2].LegendText = "Banco C";
+            this.ctBank.Series["Bank"].Points[2].SetValueXY(70, 70);
+            this.ctBank.Series["Bank"].Points[2].Label = "70%";
+
+            this.ctCreditCard.Series["CreditCard"].Points.Add(0);
+            this.ctCreditCard.Series["CreditCard"].Points[0].YValues[0] = 350;
+            this.ctCreditCard.Series["CreditCard"].Points[0].Label = "Cartão 1";
+            this.ctCreditCard.Series["CreditCard"].Points.Add(1);
+            this.ctCreditCard.Series["CreditCard"].Points[1].Label = "Cartão 2";
+            this.ctCreditCard.Series["CreditCard"].Points[1].YValues[0] = 450;
+            this.ctCreditCard.Series["CreditCard"].Points.Add(2);
+            this.ctCreditCard.Series["CreditCard"].Points[2].Label = "Cartão 3";
+            this.ctCreditCard.Series["CreditCard"].Points[2].YValues[0] = 500;
+
         }
 
         private void sairToolStripMenuItem_Click(object sender, EventArgs e)
         {
             this.Close();
         }
 
-        private void lb_MesAtual_Click(object sender, EventArgs e)
+        private void btnOpenCalendar_Click(object sender, EventArgs e)
+        {
+            FrmMothCalendar form = new FrmMothCalendar();
+            form.ShowDialog();
+            if (form.Moth != null)

```

Comment:
Faz essa verificação abaixo, pois assim, dá pra gente controlar o estado do form pelo resultado dele.
```suggestion
            if (form.ShowDialog()==DialogResult.Ok)

```
Da forma que foi implementado, o usuário pode cancelar a tela e mesmo assim será selecionado o mês.